sympy/utilities/iterables.py

```diff
@@ -19,6 +19,40 @@
     multiset_partitions_taocp, list_visitor, MultisetPartitionTraverser)
 
 
+def is_palindromic(s, i=0, j=None):
+    """return True if the sequence is the same from left to right as it
+    is from right to left in the whole sequence (default) or in the
+    Python slice ``s[i: j]``; else False.
+
+    EXAMPLES
+    ========
+
+    >>> from sympy.utilities.iterables import is_palindromic
+    >>> is_palindromic([1, 0, 1])
+    True
+    >>> is_palindromic('abcbb')
+    False
+    >>> is_palindromic('abcbb', 1)
+    False
+
+    Normal Python slicing is performed in place so there is no need to
+    create a slice of the sequence for testing:
+
+    >>> is_palindromic('abcbb', 1, -1)
+    True
+    >>> is_palindromic('abcbb', -4, -1)
+    True
+
+    See Also
+    ========
+    sympy.ntheory.digits.is_palindromic
```

Comment:
I would add a blank line above this.

Also is this a valid sphinx reference? This doesn't match the form in the style guide:
https://docs.sympy.org/latest/documentation-style-guide.html#see-also-section

Reply:
This was formerly the heading that didn't need a blank line but I can put one in. Regarding the form and style suggestions...it doesn't tell what to do to disambiguate which function you are talking about so this is how I imagine the comment-less singleton list would look like. Am open to other suggestions, however. (For anyone thinking that it should be named something else I quote "this" which says "namespaces are one honking great idea."  The path provides the disambiguation.)